tools/QuietBackgroundProcesses/DevHome.QuietBackgroundProcesses.UI/Views/SimpleTableControl.xaml

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="utf-8"?>
+<UserControl
+    x:Class="DevHome.QuietBackgroundProcesses.UI.Views.SimpleTableControl"
```

Comment:
Unless you plan to use this control somewhere else (and probably even if you are) it should probably have a more specific name.